src/psyclone/tests/test_files/dynamo0p3/1_single_invoke_kern.f90

```diff
@@ -0,0 +1,55 @@
+! -----------------------------------------------------------------------------
+! BSD 3-Clause License
```

Comment:
Do we actually need this test? I thought that `-d` option is tested elsewhere.

Reply:
I think it is right and is needed. This is used in `generator_test.py`. We are replacing the use of the `dynamo0.1` API with a `dynamo0.3` version. To test the script we need a new kernel file buried in a subdirectory of `test_files/dynamo.p3`. If it is not new then other tests fail as the same file is found more than once and an exception is raised.

Reply:
Ah, I see. Thanks for explaining!